cmd/gnmi_target/set.go

```diff
@@ -34,5 +34,16 @@ func (s *server) Set(ctx context.Context, req *pb.SetRequest) (*pb.SetResponse,
 	setResponse, err := s.Server.Set(ctx, req)
 	// Update gnmi target config store by retrieving it from the Server
 	s.configStruct, _ = s.Server.GetConfig()
+	// Prepare an update for subscribe stream and send it over update channel.
+	for _, response := range setResponse.GetResponse() {
+		update := &pb.Update{
+			Path: response.GetPath(),
+		}
+		select {
+		case s.UpdateChann <- update:
+		default:
```

Comment:
log this at least

Reply:
Sure.

Reply:
Lets keep the previous log and don't log here. I think that would be better to log when we want to send to the client. Agree?